engines/aws/app/models/aws/region.rb

```diff
@@ -19,5 +19,19 @@ def options
     def save!
       KeyValue.set(:aws_region, @value)
     end
+
+    def supported_instance_type
```

Comment:
This seems misleading. Return type is not singular based on "instance_types" name but function is "instance_type".